cmd/leanix-k8s-connector/main.go

```diff
@@ -56,21 +56,34 @@ func main() {
 	stdoutLogger, debugLogBuffer := initLogger()
 	err := parseFlags()
 	if err != nil {
-		log.Fatal(err)
+		log.Error(err)
 	}
 	enableVerbose(stdoutLogger, viper.GetBool(verboseFlag))
 
+	// use the current context in kubeconfig
+	startResponse, err := kubernetesScan(debugLogBuffer)
+	if err != nil {
+		log.Error(err)
+	}
+	log.Info("Uploading connector logs to iHub")
+	err = storage.UploadConnectorLog(startResponse.ConnectorLoggingUrl, debugLogBuffer.Bytes())
+	if err != nil {
+		log.Error(err)
+	}
+}
+
+func kubernetesScan(debugLogBuffer *bytes.Buffer) (response *leanix.SelfStartResponse, err error) {
```

Comment:
```suggestion
func KubernetesScan(debugLogBuffer *bytes.Buffer) (response *leanix.SelfStartResponse, err error) {
```
Can you please change this.